app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -51,10 +51,12 @@ def label_tag_mapping_edit
   end
 
   def entity_ui_name_or_all(entity)
-    if entity.nil?
-      _("<All>")
-    else
+    if entity
+      entity = entity.split('::').last
+      entity = 'VmOrTemplate' if entity == 'Image'
```

Comment:
@cben I don't see a way around it. Unless there's to an option to `Dictionary.gettext` that we can pass (which is being called in the `ui_lookup` method in lib/global_methods.rb), I don't see a way around this for now.